js/super_hero.js

```diff
@@ -1 +1,55 @@
 // var SuperHero = ...
+
+var Superhero = function( name, strength, loveInterest, isLoveRequited, catchphrase) {
+  this.name = name;
+  this.strength = strength;
+  this.loveInterest = loveInterest;
+  this.isLoveRequited = isLoveRequited;
+  this.catchphrase = catchphrase;
+
+  this.fightSceneAgainst = function(fightAgainst) {
+    var winFightStr = this.name + " punches " + fightAgainst + " and shouts, \"" + this.catchphrase + "\"";
+    var loseFightStr = "Oh no, " + this.name + "\'s strength is too low. " + fightAgainst + " punches them and runs off with " + this.loveInterest + ".";
```

Comment:
You might consider putting these within the `if`/`else`, so that both strings aren't created when you only need one. You might also consider moving each to its own function.